examples/restore_model_dependencies/restore_model_dependencies_example.ipynb

```diff
@@ -0,0 +1,96 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "70a2b95e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mlflow\n",
+    "from sklearn import datasets\n",
+    "from sklearn.neighbors import KNeighborsClassifier\n",
+    "from pathlib import Path\n",
+    "\n",
+    "X, y = datasets.load_iris(as_frame=True, return_X_y=True)\n",
+    "model = KNeighborsClassifier()\n",
+    "model.fit(X, y)\n",
+    "\n",
+    "model_path = \"/tmp/sk_model_01\"\n",
+    "\n",
+    "mlflow.sklearn.save_model(model, model_path)\n",
+    "\n",
+    "model_requirements_file_path = mlflow.pyfunc.get_model_dependencies(model_path)"
```

Comment:
I ran this line on my machine and got the following result:

```python
>>> import mlflow.pyfunc
>>> mlflow.pyfunc.get_model_dependencies("runs:/4d2f0c09525742579b0382754a73017b/model")
2022/04/22 12:13:51 INFO mlflow.pyfunc: To install these model dependencies, run the following command: 'pip install -r /home/haru/Desktop/repositories/mlflow/mlruns/0/4d2f0c09525742579b0382754a73017b/artifacts/model/requirements.txt'.
```

and thought `what do "these model dependencies" refer to?`

Reply:
What about rephrasing it to be :
```
To install python packages of the exact versions that was used to build the model, ...
```

Reply:
Or we could say:

```
To install the dependencies that were used to train the model, run ...
```